<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Продукционный метод</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Пациент</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="patientComboBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Болезнь:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editDiseases">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Лечение:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="editTreatment">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="isTrauma">
            <property name="text">
             <string>Трамвы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="isMental">
            <property name="text">
             <string>Психическое</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="isVirus">
            <property name="text">
             <string>Вирусы</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSolutionForward">
          <property name="text">
           <string>Решение прямое</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxkindThreatment"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Список пациентов:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="editListPatients">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSolutionBackward">
          <property name="text">
           <string>Решение обратное</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Симптомы:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Температура</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editTemperature">
            <property name="readOnly">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="isCough">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Кашель</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isHeadache">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Болит голова</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isStomachAche">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Болит живот</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isRush">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Сыпь</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isAggression">
          <property name="text">
           <string>Агрессия</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isNerv">
          <property name="text">
           <string>Нервозность</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isStra">
          <property name="text">
           <string>Страх</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isBlooding">
          <property name="text">
           <string>Кровоточащая рана</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isVosp">
          <property name="text">
           <string>Воспаление</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isLowweight">
          <property name="text">
           <string>Низкий вес</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="isIszo">
          <property name="text">
           <string>Изжога</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
